.github/dependabot.yml

```diff
@@ -58,155 +58,14 @@ updates:
 
 # Maintain dependencies for Golang
 - package-ecosystem: "gomod"
-  directory: "/src/go/docker-credential-none"
+  directories:
+  - "/src/go/*"
```

Comment:
I think this pattern matches just a subset of the patterns on the next line, and would then be redundant. But it doesn't hurt either.